app/views/idv/doc_auth/front_image.html.erb

```diff
@@ -4,8 +4,8 @@
 
 <%= render 'idv/doc_auth/front_of_state_id_image' %>
 
-<h1 class='h3 my0'>
-  <%= t('doc_auth.headings.upload_front') %>
+<h1 class='h3 my0' for='doc_auth_image'>
```

Comment:
I don't think the `for` is intended to be here? Possible typo since it's on the following line (appropriately) as well.

```suggestion
<h1 class='h3 my0'>
```

Reply:
Yep, thank you!